crates/re_renderer/shader/device_info.wgsl

```diff
@@ -0,0 +1,3 @@
+// True if we're running on Gles/WebGL.
+// (patched accordingly if necessary)
+const GLES = false;
```

Comment:
I think something like this would be better:

```suggestion
// True if we're running on Gles/WebGL.
// Will be set by `re_renderer` during startup
const GLES = ${GLES};
```

That way we cannot accidentally break the patch-code without us noticing it.

…or just pass it in the global uniform buffer